src/site/content/en/learn/css/pseudo-elements/pseudo-elements.assess.yml

```
@@ -0,0 +1,33 @@
+setLeader: Test your knowledge of pseudo elements
+height: unset
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1+"
+    correctAnswers: "1,4,5"
+    stem: Which of the following are not pseudo elements?
+    options:
+      - content: "`::before`"
+        rationale: "Don't forget to add `content: '';`"
+      - content: "`::first-paragraph`"
+        rationale: "This doesn't exist in CSS"
+      - content: "`::after`"
+        rationale: "Don't forget to add `content: '';`"
+      - content: "`::marker`"
+        rationale: "This is the bullet element when you use a list element or
+        display type"
+      - content: "`::pencil`"
+        rationale: "This doesn't exist in CSS"
+      - content: "`:active`"
+        rationale: "This is a pseudo class not a pseudo element"
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "1"
+    stem: Psuedo elements can be found in the HTML
```

I'm not sure I understand what this means. Do you mean "can be seen when you view source?" "Exist in the HTML document that you created".

(it is very very common that devs don't understand that the DOM they see in DevTools is different to what they get if they view source... )

---

"Exists in the HTML document you created or are looking inside of"

it def is trying to poke this grey space, to harden that these "elements" aren't in the document, in case they hadnt realized it fully yet.

```suggestion
    stem: Psuedo elements can be found in an HTML file
```

thoughts on that adjustment?